packages/arb-util/configuration/configuration.go

```diff
@@ -261,25 +276,59 @@ func Parse(ctx context.Context) (*Config, *Wallet, *ethutils.RPCEthClient, *big.
 	}
 
 	// Fixup directories
-	if len(out.Persistent.Storage.Path) == 0 {
-		// Error message will be output by caller
-		return out, nil, nil, nil, nil
+	homeDir, err := os.UserHomeDir()
+	if err != nil {
+		return nil, nil, nil, nil, errors.Wrap(err, "Unable to read users home directory")
 	}
 
-	if len(out.Persistent.Database.Path) == 0 {
-		out.Persistent.Database.Path = path.Join(out.Persistent.Storage.Path, "arbStorage")
+	// Make persistent storage directory relative to home directory if not already absolute
+	out.Persistent.Storage.Path = path.Join(homeDir, out.Persistent.Storage.Path)
```

Comment:
Doesn't this force the path to always be in the home directory? I'm on board with using `~/.arbitrum` as a default, but I don't think we want to only support relative paths to the home dir since that seems unnecessarily restrictive

Reply:
I mistakenly assumed path.Join would do the right thing.  Updated to allow absolute paths